fragment_internal_test.go

```diff
@@ -37,12 +39,17 @@ var (
 	// In order to generate the sample fragment file,
 	// run an import and copy PILOSA_DATA_DIR/INDEX_NAME/FRAME_NAME/0 to testdata/sample_view
 	FragmentPath = flag.String("fragment", "testdata/sample_view/0", "fragment path")
+	TempDir      = ""
 )
 
+func init() {
+	flag.StringVar(&TempDir, "temp-dir", "", "Directory in which to place temporary data (e.g. for benchmarking). Useful if you are trying to benchmark different storage configurations.")
+}
+
 // Ensure a fragment can set a bit and retrieve it.
 func TestFragment_SetBit(t *testing.T) {
 	f := mustOpenFragment("i", "f", viewStandard, 0, "")
-	defer f.Close()
+	defer f.Clean()
```

Comment:
Looking at Clean, I can't see why it would happen, but what does happen if the clean is not fully successful? It looks like the result would be fragments already existing, and possibly affecting timing. So possibly we *should* be watching for that. But it seems like it shouldn't be possible under normal circumstances. Still, maybe some kind of sanity-check for this?

Possibly a helper function which takes an f and a t, and checks the error of f.Clean and if it gets one, calls t.Fail?